compiler/rustc_borrowck/src/region_infer/opaque_types.rs

```diff
@@ -372,38 +347,33 @@ fn check_opaque_type_well_formed<'tcx>(
     }
 }
 
-fn check_opaque_type_parameter_valid(
-    tcx: TyCtxt<'_>,
-    opaque_type_key: OpaqueTypeKey<'_>,
+fn check_opaque_type_parameter_valid<'tcx>(
```

Comment:
Linking to where these opaque type parameter validity rules are documented would be useful here.